go.mod

```diff
@@ -82,9 +83,14 @@ require (
 	github.com/blakesmith/ar v0.0.0-20190502131153-809d4375e1fb // indirect
 	github.com/bombsimon/wsl/v3 v3.1.0 // indirect
 	github.com/cavaliercoder/go-cpio v0.0.0-20180626203310-925f9528c45e // indirect
+	github.com/containerd/stargz-snapshotter/estargz v0.10.0 // indirect
 	github.com/daixiang0/gci v0.2.7 // indirect
 	github.com/davecgh/go-spew v1.1.1 // indirect
 	github.com/denis-tingajkin/go-header v0.3.1 // indirect
+	github.com/docker/cli v20.10.10+incompatible // indirect
+	github.com/docker/distribution v2.7.1+incompatible // indirect
+	github.com/docker/docker v20.10.10+incompatible // indirect
```

Comment:
I don't think this cmd/mkctr tool really needs to live in tailscale/tailscale. How about a new repo at github.com/tailscale/mkctr or something?

Reply:
yeah i was considering that too, but i thought getting initial thoughts here would be easier

Reply:
https://github.com/tailscale/mkctr/pull/1